news/2022-06-11-new-featured-artist-kanpyohgo.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: Kanpyohgo"
+date: 2022-06-11 13:00:00 +0000
+---
+
+**Kanpyogho** shuffles into our deck of Featured Artists!
+
+![](https://assets.ppy.sh/artists/274/header.jpg)
+
+A professional in both hardware and sound engineering, **Kanpyohgo** (a.k.a. *Kanpyo*) is one of the few people whose love for waveforms includes both audio and electric circuitry.
+
+Avid circle clickers may recognize **Kanpyohgo** from the stream-jump focused beatmap [Unmei no Dark Side -Rolling Gothic mix-](https://osu.ppy.sh/beatmapsets/514980), having amassed over a thousand favourites and approaching 2 million plays since reaching ranked!
```

Comment:
```suggestion
Avid circle clickers may recognize **Kanpyohgo** from the stream jump–focused beatmap [Unmei no Dark Side -Rolling Gothic mix-](https://osu.ppy.sh/beatmapsets/514980), having amassed over a thousand favourites and approaching 2 million plays since reaching ranked!
```